test/modules/LinearBoltzmannSolvers/Transport_Keigen/c5g7/mesh/gmesh_refined.lua

```diff
@@ -0,0 +1,18 @@
+--############################################### Setup mesh
+chiMeshHandlerCreate()
+
+umesh = chiUnpartitionedMeshFromMshFormat("mesh/2D_c5g7_refined.msh",true)
+
+--############################################### Create meshers
+chiSurfaceMesherCreate(SURFACEMESHER_PREDEFINED);
+chiVolumeMesherCreate(VOLUMEMESHER_UNPARTITIONED,umesh);
+
+--chiVolumeMesherSetKBAPartitioningPxPyPz(2,2,1)
+--chiVolumeMesherSetKBACutsX({27.72})
+--chiVolumeMesherSetKBACutsY({27.72})
+--chiVolumeMesherSetProperty(PARTITION_TYPE,KBA_STYLE_XYZ)
```

Comment:
Remove unless there is a compelling reason to keep.

Reply:
Deleted